.github/workflows/docs.yml

```diff
@@ -12,17 +11,16 @@ jobs:
     name: Publish
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
 
-    - name: Use Node.js
-      uses: actions/setup-node@v3
+    - uses: actions/setup-node@v3
 
-    - name: Build documentation
-      run: make docs-build
+    - run: make docs-build
+    
+    - run: touch ./build/docs/.nojekyll
```

Comment:
One of the storybook files has a name that starts with an underscore and [something about Jekyll was preventing that file from working](https://github.com/storybookjs/storybook/issues/21771).